### PR TITLE
Add releases to readme

### PR DESCRIPTION
**Pull request by jignaciopm**
This PR aims to add information on how to release this repository

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -93,3 +93,30 @@ In order to activate these processor some configuratios are required.
...
+- Once the PR is approved, we perform squash and merge.
+
+To release this repository:
+
+1. In development, we position ourselves in the main branch and update it. 
+2. `bumpversion`_: Then we write in console::
+
+    bumpversion [major, minor, patch]
+3. The changes that are in 'unreleased' become 'released', they are identified with the date and the version that has generated `bumpversion`_.
+4. The commit generated by `bumpversion`_ is pushed, which must also include the recent change in `CHANGELOG.rst`_.
```
Comment (mariajgrimaldi) [inline README.rst]: what about moving the changes from 'unreleased'?

**Review by MoisesGSalas — COMMENTED** [review]
I think a good place to have this would be a `CONTRIBUTING.rst` file. See the [github docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)

File: README.rst (hunk 1 above)
Comment (jignaciopm) [reply README.rst]: Do you mean when is the commit done? As I see it, bumpversion is done, then 'unreleased' is moved to 'released', and that commit is squashed with the bumpversion to upload a single commit. 

@andrey-canon Can you confirm if this is so?

Comment (mariajgrimaldi) [reply README.rst]: I think the `that commit is squashed with the bumpversion to upload a single commit.` is not necessary

Comment (jignaciopm) [reply README.rst]: @mariajgrimaldi I understand. You are probably correct. But how do you suggest these kinds of confirmations be made? [https://github.com/eduNEXT/eox-audit-model/commit/201a982f2d1907c62daed0377e4869ce8eb04797](https://github.com/eduNEXT/eox-audit-model/commit/201a982f2d1907c62daed0377e4869ce8eb04797)

Comment (mariajgrimaldi) [reply README.rst]: Great. Then what you said is fine

**Review by mariajgrimaldi — APPROVED** [review]

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**